### PR TITLE
Consistent Yaml indentation size

### PR DESCRIPTION
**Pull request by simara-svatopluk**
| Q             | A
| ------------- | ---
|Description, reason for the PR| #205
|New feature| Yes
|BC breaks| No
|Fixes issues| #205
|Standards and tests pass| Yes
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -24,11 +24,16 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
+#### Changed
+- [#389 - Consistent Yaml indentation size](https://github.com/shopsys/shopsys/pull/389)
+    - standardized yaml files indentation to 4 spaces
+
 #### Fixed
 - [#260 - JS validation: dynamically added form inputs are now validated](https://github.com/shopsys/shopsys/pull/260)
 
 ## [7.0.0-alpha4] - 2018-08-02
 ### [shopsys/framework]
+
```
Comment (PetrHeinz) [inline CHANGELOG.md]: excessive whitespace

File: CHANGELOG.md [hunk 2]
```diff
@@ -24,11 +24,16 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 ### [shopsys/framework]
+#### Changed
+- [#389 - Consistent Yaml indentation size](https://github.com/shopsys/shopsys/pull/389)
```
Comment (PetrHeinz) [inline CHANGELOG.md]: This wasn't just a change in the shopsys/framework but in all packages.

**Comment by simara-svatopluk** [comment]
Fixed.

**Comment by simara-svatopluk** [comment]
I tried to find some fixes, but since this is a bug, I didn't spent much time on it. I can create an US that will create sniffs for indentantion, what do You think?

**Comment by simara-svatopluk** [comment]
I'll fix commit message when CR is done

File: CHANGELOG.md [hunk 3]
```diff
@@ -24,9 +24,63 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
+
+### [shopsys/product-feed-zbozi]
+#### Changed
+- [#389 - Consistent Yaml indentation size](https://github.com/shopsys/shopsys/pull/389)
+    - standardized yaml files indentation to 4 spaces
+
+### [shopsys/project-base]
+#### Changed
+- [#389 - Consistent Yaml indentation size](https://github.com/shopsys/shopsys/pull/389)
+    - standardized yaml files indentation to 4 spaces
```
Comment (PetrHeinz) [inline CHANGELOG.md]: I consider having the same message about whitespace changes copy-pasted in the CHANGELOG.md eleven times ridiculous, to be honest. Can we do it some other way? Maybe mention it just in shopsys/shopsys?

Comment (simara-svatopluk) [reply CHANGELOG.md]: Well, this is a valid point, but copy-paste is the way we do it when the change includes more packages. Try to run `cat CHANGELOG.md | sort` and You'll see it is normal

File: project-base/app/config/config_dev.yml [hunk 4]
```diff
@@ -1,43 +1,43 @@
...
-  intercept_redirects: false
+    toolbar: true
+    intercept_redirects: false
 
 debug:
-  max_items: 500
-  max_string_length: 5000
\ No newline at end of file
+    max_items: 500
+    max_string_length: 5000
```
[the remark this answers is not in the document]
Comment (PetrHeinz) [reply project-base/app/config/config_dev.yml]: You can search for `[^\n]\z` in `*.yml`s using PhpStorm, it'll found 3 ocurrences.

**Pull request merged**